cachi2/core/package_managers/pip.py

```diff
@@ -1289,7 +1289,7 @@ def from_line(cls, line: str, options: list[str]) -> Optional["PipRequirement"]:
         :return: PipRequirement instance
         """
         to_be_parsed = line
-        qualifiers = {}
+        qualifiers: dict[str, str]  = {}
```

Comment:
flake8 ` E221: multiple spaces before operator`

Reply:
Also, the constructor for this class defines the attribute as `self.qualifiers: dict[str, Any] = {}`, so we should be consistent and change the instance attribute to a more stricter type to follow the rest of the module.